### PR TITLE
WT-8583 Remove commit_timestamp config from set_timestamp

### PR DESCRIPTION
**Pull request by fionaochee**

### CODE REVIEW

**Review by etienneptl — APPROVED** [review]
Excellent work @fionaochee, LGTM!

File: test/suite/test_timestamp02.py [hunk 1]
```diff
@@ -101,11 +101,11 @@ def test_basic(self):
 
         # Everything up to and including timestamp 200 has been committed.
         self.assertTimestampsEqual(self.conn.query_timestamp(), self.timestamp_str(200))
-
-        # Test that we can manually move the commit timestamp back
-        self.conn.set_timestamp('commit_timestamp=' + self.timestamp_str(150))
+
```
Comment (etienneptl) [inline test/suite/test_timestamp02.py:104]: nitpick: are those extra white spaces?

**Review by etienneptl — APPROVED** [review]
LGTM, good job!

**Review by keithbostic — APPROVED** [review]
lgtm -- great cleanup, thanks!

**Review by jiechenbo — APPROVED** [review]
LGTM!

**Pull request merged**